src/ingest_llm_as/services/omegakg_client.py

```
@@ -0,0 +1,130 @@
+                    json=digest
+                )
+                
+                # Success or validation rejection (don't retry)
+                # 200: Handled success/rejection/duplicate
+                # 409: Conflict/Duplicate (if the API chooses to return 409 instead of 200)
+                if response.status_code in [200, 409]:
+                    return response.json()
+                
+                # Authentication error (don't retry)
+                if response.status_code == 401:
+                    logger.error(f"OmegaKG API authentication failed: {response.text}")
```

Ruff (0.14.11)

Logging statement uses f-string

(G004)